crates/iota-indexer/src/types.rs

```diff
@@ -427,13 +428,45 @@ impl IndexedObject {
     }
 }
 
+#[cfg(any(feature = "pg_integration", feature = "shared_test_runtime", test))]
+impl IndexedObject {
+    pub fn random() -> Self {
```

Comment:
nit: could we maybe name it `random_for_testing_only`, analogous to the  method name in `IotaAddress`? Since I believe it should not be used in prod code

Reply:
Okay, now I see that it's only compiled for test feature flags.